subworkflows/nf-core/fastq_align_chromap/meta.yml

```diff
@@ -0,0 +1,61 @@
+name: "fastq_align_chromap"
+description: Align reads to a reference genome using bowtie2 then sort with samtools
+keywords:
+  - align
+  - fasta
+  - genome
+  - reference
+modules:
+  - chromap/chromap
+  - samtools/sort
+  - samtools/index
+  - samtools/stats
+  - samtools/idxstats
+  - samtools/flagstat
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - reads:
+      type: file
+      description: |
+        List of input FastQ files of size 1 and 2 for single-end and paired-end data,
+        respectively.
+  - index:
```

Comment:
Missing FASTA here

Maybe also describe the required input channel structure?

Reply:
Addressed here: 10dd532c63090e1f34ca2d67bf750dc25a145c60